Neos.Flow/Classes/Persistence/Doctrine/Service.php

```diff
@@ -217,7 +217,7 @@ public function runDql(string $dql, int $hydrationMode = \Doctrine\ORM\Query::HY
      */
     protected function getDependencyFactory(): DependencyFactory
     {
-        $migrationsPath = Files::concatenatePaths([FLOW_PATH_DATA, 'DoctrineMigrations']);
+        $migrationsPath = Files::concatenatePaths([FLOW_PATH_DATA, 'Temporary', 'DoctrineMigrations']);
```

Comment:
```suggestion
        $migrationsPath = Files::concatenatePaths([FLOW_PATH_TEMPORARY, 'DoctrineMigrations']);
```

Reply:
IIRC `FLOW_PATH_TEMPORARY` already takes `Environment::getPathToTemporaryDirectory()` into account. So +1 for changing the PR as suggested.

The "allow FLOW_PATH_DATA to come from env"-topic should be moved to a separate issue IMO